packages/itwin/tree-widget/src/components/trees/stateless/common/components/TreeNodeRenderer.tsx

```diff
@@ -19,11 +19,11 @@ type CoreTreeNodeRendererProps = ComponentPropsWithoutRef<typeof CoreTreeNodeRen
 type TreeNodeRendererProps = CoreTreeNodeRendererProps & TreeNodeRendererOwnProps;
 
 /** @internal */
-export function TreeNodeRenderer({ checkboxProps, ...props }: TreeNodeRendererProps) {
+export function TreeNodeRenderer({ checkboxProps, nodeProps, ...props }: TreeNodeRendererProps) {
   return (
     <CoreTreeNodeRenderer
       {...props}
-      nodeProps={{ className: cx("tw-tree-node", props.isSelected && "selected", props.className) }}
+      nodeProps={{ ...nodeProps, className: cx("tw-tree-node", props.isSelected && "selected", props.className) }}
```

Comment:
I guess `...nodeProps` can be removed? Or if you expect that we will need to pass custom nodeProps in the future same need to be done with `contentProps`

Reply:
Removed, we can add it when we need it.